R/table-object.R

```diff
@@ -921,6 +922,33 @@ st_edit <- function(x, ...) {
   x
 }
 
+#' Set table caption
+#'
+#' @param x an stobject.
+#' @param text caption text to be parsed.
+#' @param ... additional arguments passed to [as.caption()].
+#'
+#' @details
+#' A short title can also be specified by including it in brackets (`[]`) as
```

Comment:
This "also" is referring, I think, to the fact that you can pass `short` via `...` to `as.caption`.  But I don't expect that to be obvious to a reader given `short` won't appear in st_caption's documentation.  I think it'd be good to explicitly mention the `short` option.

Reply:
I see. Thanks that's right. I think `short` used to be an argument to `st_caption()` but I moved it farther in. I'll get better documentation here.